tests/benchmark-runner-tests.mjs

```diff
@@ -138,8 +138,8 @@ describe("BenchmarkRunner", () => {
 
             it("should remove the previous frame and then the current frame", () => {
                 assert.calledTwice(_loadFrameStub);
-                assert.calledOnce(_appendFrameStub);
-                assert.calledTwice(_removeFrameStub);
+                assert.calledTwice(_appendFrameStub);
+                assert.calledThrice(_removeFrameStub);
```

Comment:
I believe `removeFrame` is called once more because it's called by `finishARunllSuites` currently, but that it will be the more expected `twice` after my suggested change above.